doc/nrf/libraries/nrf_security/doc/backend_config.rst

```diff
@@ -481,13 +481,29 @@ To configure Rivest-Shamir-Adleman (RSA) support, set the :kconfig:option:`CONFI
 Feature support
 ===============
 
-+-----------+-------------------+----------------------+-----------------------+
-| Algorithm | Backend           | Key size             | Note                  |
-+===========+===================+======================+=======================+
-| RSA       | cc3xx             | Limited to 2048 bits |                       |
-|           +-------------------+----------------------+-----------------------+
-|           | nrf_oberon        | N/A                  | Backend not supported |
-+-----------+-------------------+----------------------+-----------------------+
++-----------+-------------------+--------------+
+| Algorithm | Backend           | Key size     |
++===========+===================+==============+
+| RSA       | cc310             | 1024-bit key |
+|           |                   +--------------+
+|           |                   | 1536-bit key |
+|           |                   +--------------+
+|           |                   | 2048-bit key |
+|           +-------------------+--------------+
+|           | cc312             | 1024-bit key |
+|           |                   +--------------+
+|           |                   | 1536-bit key |
+|           |                   +--------------+
+|           |                   | 2048-bit key |
+|           |                   +--------------+
+|           |                   | 3072-bit key |
+|           +-------------------+--------------+
+|           | nrf_oberon        | 1536-bit key |
```

Comment:
From what I can tell nrf_oberon supports 1024-bit key also.

Reply:
right. added that. it also support 4k, 6k and 8k but that we have not tested that.

Reply:
True, but from 4k and up it requires a larger stack size in TF-M, so we need to do some changes to support these key sizes.